### ISSUE TITLE
Tests may fail in shardingsphere-jdbc-governance

### ISSUE THREAD

**Issue by TeslaCN** [apache/shardingsphere#11873]
## Bug Report

Refer to https://github.com/apache/shardingsphere/pull/11872/checks?check_run_id=3357183318

```
[INFO] Running org.apache.shardingsphere.driver.governance.internal.datasource.GovernanceShardingSphereDataSourceTest
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.032 s <<< FAILURE! - in org.apache.shardingsphere.driver.governance.internal.datasource.GovernanceShardingSphereDataSourceTest
Error:  assertInitializeGovernanceShardingSphereDataSource(org.apache.shardingsphere.driver.governance.internal.datasource.GovernanceShardingSphereDataSourceTest)  Time elapsed: 0.031 s  <<< ERROR!
java.lang.NullPointerException
	at org.apache.shardingsphere.driver.governance.internal.datasource.GovernanceShardingSphereDataSourceTest.assertInitializeGovernanceShardingSphereDataSource(GovernanceShardingSphereDataSourceTest.java:36)

[INFO] Running org.apache.shardingsphere.driver.governance.internal.state.DriverStateContextTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.617 s - in org.apache.shardingsphere.driver.governance.internal.state.DriverStateContextTest
[INFO] Running org.apache.shardingsphere.driver.governance.internal.state.impl.OKDriverStateTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in org.apache.shardingsphere.driver.governance.internal.state.impl.OKDriverStateTest
[INFO] Running org.apache.shardingsphere.driver.governance.internal.state.impl.CircuitBreakDriverStateTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in org.apache.shardingsphere.driver.governance.internal.state.impl.CircuitBreakDriverStateTest
[INFO] Running org.apache.shardingsphere.driver.governance.internal.circuit.datasource.CircuitBreakerDataSourceTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in org.apache.shardingsphere.driver.governance.internal.circuit.datasource.CircuitBreakerDataSourceTest
[INFO] Running org.apache.shardingsphere.driver.governance.internal.circuit.metadata.CircuitBreakerDatabaseMetaDataTest
[INFO] Tests run: 176, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.115 s - in org.apache.shardingsphere.driver.governance.internal.circuit.metadata.CircuitBreakerDatabaseMetaDataTest
[INFO] Running org.apache.shardingsphere.driver.governance.internal.circuit.connection.CircuitBreakerConnectionTest
[INFO] Tests run: 17, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.018 s - in org.apache.shardingsphere.driver.governance.internal.circuit.connection.CircuitBreakerConnectionTest
[INFO] 
[INFO] Results:
[INFO] 
Error:  Errors: 
Error:    GovernanceShardingSphereDataSourceTest.assertInitializeGovernanceShardingSphereDataSource:36 » NullPointer
[INFO] 
Error:  Tests run: 223, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by menghaoranss** [comment]
```
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.033 s <<< FAILURE! - in org.apache.shardingsphere.driver.governance.internal.datasource.GovernanceShardingSphereDataSourceTest
22086
Error:  assertInitializeGovernanceShardingSphereDataSource(org.apache.shardingsphere.driver.governance.internal.datasource.GovernanceShardingSphereDataSourceTest)  Time elapsed: 0.033 s  <<< ERROR!
22087
java.lang.NullPointerException
22088
	at org.apache.shardingsphere.driver.governance.internal.datasource.GovernanceShardingSphereDataSourceTest.assertInitializeGovernanceShardingSphereDataSource(GovernanceShardingSphereDataSourceTest.java:36)
```

**Comment by menghaoranss** [comment]
Fixed by #11885
